src/bootstrap/bootstrap.py

```diff
@@ -48,44 +48,45 @@ def eprint(*args, **kwargs):
     print(*args, **kwargs)
 
 
-def get(base, url, path, checksums, verbose=False):
+def get(base, url, path, checksums, verbose=False, verify_checksum=True):
     with tempfile.NamedTemporaryFile(delete=False) as temp_file:
         temp_path = temp_file.name
-
     try:
-        if url not in checksums:
+        if url not in checksums and verify_checksum:
             raise RuntimeError(
                 (
-                    "src/stage0 doesn't contain a checksum for {}. "
+                    "src/stage0.json doesn't contain a checksum for {}. "
```

Comment:
There is no such file

```suggestion
                    "src/stage0 doesn't contain a checksum for {}. "
```